h2o-r/h2o-package/R/models.R

```diff
@@ -302,7 +302,7 @@ h2o.getFutureModel <- function(object, verbose=FALSE) {
       } else {
         if (!inherits(paramValue, type)) {
           e <- paste0(e, "\"", name , "\" must be of type ", type, ", but got ", class(paramValue), ".\n")
-        } else if ((length(paramDef$values) > 1L) && !(paramValue %in% paramDef$values)) {
+        } else if ((length(paramDef$values) > 1L) && (is.null(paramValue) || !(tolower(paramValue) %in% tolower(paramDef$values)))) {
```

Comment:
for models validation, default values are retrieved from backend: therefore needs to apply case-insensitive validation at lower level. This will apply for all enums used as model parameters.